src/System.IO.Pipelines.Networking.Sockets/SocketListener.cs

```diff
@@ -137,22 +137,19 @@ private void OnAccept(SocketAsyncEventArgs e)
             // will cause a stack-dive in the sync (backlog) case
         }
 
-        private async void ExecuteConnection(SocketConnection conn)
+        private async void ExecuteConnection(SocketConnection connection)
```

Comment:
You can catch/report/log on `var ignore =` exceptions with the `TaskScheduler.UnobservedTaskException` handler, and in 4.5+ they don't cause an app crash. `async void` exceptions cut through the app like butter and bring it all down.